src/components/menu/menu.ts

```diff
@@ -30,13 +30,12 @@ export default class SlMenu extends ShoelaceElement {
   }
 
   private handleClick(event: MouseEvent) {
-    const target = event.target as HTMLElement;
-    const item = target.closest('sl-menu-item');
-
-    if (!item || item.disabled || item.inert) {
+    if (!(event.target instanceof SlMenuItem)) {
```

Comment:
I know previously we used `target.closest('sl-menu-item')` so this isn't a huge deal, but we should think of defining common interfaces for cases like this so we don't need to check instance types.

This is purely a comment for the future, and shouldn't affect this PR.